README.md

```diff
@@ -1,37 +1,53 @@
 # Wormhole Solidity SDK
 
-The purpose of this SDK is to provide helpers to take your existing single-chain solidity application cross-chain using Wormhole's automatic relayers
+The purpose of this SDK is to make integrations with Wormhole on EVM compatible chains as smooth as possible by providing all necessary Solidity interfaces along with useful libraries and tools for testing.
 
-### Installation
+This SDK was originally created for integrations with the WormholeRelayer and then expanded to cover all integrations.
+
+## Installation
 
 **Foundry and Forge**
 
 ```bash
 forge install wormhole-foundation/wormhole-solidity-sdk
 ```
 
-### Example Usage + Introduction to Automatic Relayers
+**Solc Version**
+
+Currently the SDK uses solc version 0.8.19 to avoid issues with OPCODE0 which was introduced in 0.8.20 but which is not supported on many EVM chains.
+
+## WormholeRelayer
+
+### Introduction
+
+The WormholeRelayer (also sometimes referred to as the automatic or generic relayer) allows integrators to leverage external parties known as delivery providers, to relay messages emitted on a given source chain to the intended target chain.
+
+This frees integrators, who are building a cross-chain app, from the cumbersome and painful task of having to run relaying infrastructure themselves, without off-loading the burden of having to submit any transactions on the target chain themselves (and thus e.g. dealing with the headache of having to acquire gas tokens for the target chain).
```

Comment:
```suggestion
This frees integrators, who are building a cross-chain app, from the cumbersome and painful task of having to run relaying infrastructure themselves (and thus e.g. dealing with the headache of having to acquire gas tokens for the target chain).
```

Reply:
even simpler:
```suggestion
This frees cross-chain app developers from having to run infrastructure that submits transactions on the destination chain (and thus e.g. having to maintain a gas token balance on that chain).
```

Reply:
If my writing was only half as elegant and concise as my code...